Makefile

```diff
@@ -11,8 +13,17 @@ TARGETS := $(shell ls scripts)
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
+cleanup: .dapper
+	./.dapper -m bind $(SCRIPTS_DIR)/cleanup.sh
+
+clusters: build
```

Comment:
don't we need package after build too?

Reply:
oh as you clarified we don't have a separate action in the submariner-operator repo